src/expr/src/vector_op/agg/aggregator.rs

```diff
@@ -268,7 +268,18 @@ pub fn create_agg_state_unary(
         (SingleValue, SingleValue::new(), float64, float64, None),
         (SingleValue, SingleValue::new(), decimal, decimal, None),
         (SingleValue, SingleValue::new(), boolean, boolean, None),
+        (SingleValue, SingleValue::new(), date, date, None),
+        (SingleValue, SingleValue::new(), timestamp, timestamp, None),
+        (SingleValue, SingleValue::new(), time, time, None),
```

Comment:
FYI there are 2 more types: interval and timestampz
https://www.risingwave.dev/docs/latest/sql-data-types/